package.json

```diff
@@ -334,20 +328,52 @@
             "description": "Per-workspace list of module directories for the language server to read",
             "deprecationMessage": "Deprecated: This setting is not used by the extension and will be removed in a future release"
           },
+          "terraform.languageServer.excludeRootModules": {
+            "scope": "resource",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "default": [],
+            "description": "Per-workspace list of module directories for the language server to exclude",
+            "deprecationMessage": "Deprecated: This setting is not used by the extension and will be removed in a future release"
+          },
+          "terraform.languageServer.ignoreDirectoryNames": {
+            "scope": "resource",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "default": [],
+            "description": "Per-workspace list of directory names for the language server to ignore when indexing files"
+          },
+          "terraform-ls.rootModules": {
+            "scope": "resource",
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "default": [],
+            "description": "Per-workspace list of module directories for the language server to read",
+            "deprecationMessage": "Moved: Use `#terraform.languageServer.rootModules#` instead"
```

Comment:
It would show a link to click if i had used the markdown version of it. Fixing!

Reply:
It still looks the same, but it may be just my local environment - I couldn't get the markdown to work before when toying with deprecation messages at all.

Reply:
What version are you on?

Insiders:

![image](https://user-images.githubusercontent.com/272569/176448266-4d70ea28-b19f-447d-8dce-e493861d4661.png)

Stable:

![image](https://user-images.githubusercontent.com/272569/176448572-7da8dc1d-82e3-4991-aaa1-d55055f53785.png)